pkg/storage/internalstorage/register.go

```diff
@@ -90,9 +91,17 @@ func newLogger(cfg *Config) (logger.Interface, error) {
 			return nil, fmt.Errorf("mkdir log dir error: %w", err)
 		}
 
-		logWriter, err = os.OpenFile("/var/log/clusterpedia/internalstorage.log", os.O_APPEND|os.O_WRONLY|os.O_CREATE, 0644)
+		path := "/var/log/clusterpedia/internalstorage.log"
+		f, err := os.OpenFile(path, os.O_APPEND|os.O_WRONLY|os.O_CREATE, 0644)
 		if err != nil {
-			return nil, fmt.Errorf("open storage log file error: %w", err)
+			return nil, fmt.Errorf("open storage log file %s error: %w", path, err)
+		}
+		f.Close()
```

Comment:
Data cannot be written after the file is closed.

Reply:
> Data cannot be written after the file is closed.

This is only an ensureFile logic, lumberjack.Logger would open file of path later.

Reply:
lumberjack will create `dir` and `filename`, this should be a capability of lumberjack, we don't need to create additional